File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/YouCannotLowerYourself.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class YouCannotLowerYourself : StoryChoice
	{
		internal YouCannotLowerYourself() : base(StoryChoice.GetLocalizedString("ch5YouCannotLowerYourself"), 5025)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5AnsweredLiteratureStory = LiteratureStoryAnswer.PlayToStrengths;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/YouCannotLowerYourself.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.LiteratureStoryAnswer;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class YouCannotLowerYourself extends StoryChoice {
    public YouCannotLowerYourself() {
        super(R.string.ch5YouCannotLowerYourself, 5025);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5AnsweredLiteratureStory(LiteratureStoryAnswer.PlayToStrengths);
        super.select();
    }
}
